i18n-guidelines.md

```diff
@@ -1,24 +1,96 @@
-* **Review PRs and translations.** New to reviewing GitHub PRs and translations? No worries, we have you covered in our [Reviewer Process Guide](#reviewer-process-guide).
+## How Can I Contribute?
+
+There are two types of i18n contributions that we accept:
+
+1. Translate our Intro to Open Source course.
+2. Review pull requests (PRs) and translations.
+
+### Translate the Intro to Open Source Course
+
+We have two types of translations:
+
+#### 1. Official Translations
+
+Official translations start as a post on our [discussion board](https://github.com/open-sauced/intro/discussions). If there is enough interest and volunteers, we can add the official translation as an option to view within our README.
+
+#### 2. Community Translations
+
+We can’t always support the maintenance of translations. However, we know that some contributors are willing to translate. For that reason, we have our Community Translations section. If you've forked the repository and created a translation, you can add a link to your forked translation to the `community-translations.md` file.
+
+We encourage you to add it to the [discussion board](https://github.com/open-sauced/intro/discussions) as well. We will consider moving it to an official translation if it becomes popular and has enough support.
+
+### Review Pull Requests (PRs) and Translations
+
+If you are familiar with the translated language(s), you can help us review the translations and the PRs. Please head over to our [Reviewer Process Guide](#reviewer-process-guide) for more information.
+
+## Adding Translations
+
+First, please read our [Contributing Guide](CONTRIBUTING.md) to setting up the project locally and for the technical instruction. Then, follow these steps to add the translations:
+
+1. **Identify target languages**.
+
+   Determine which languages you want to add translations for. Make sure these languages are relevant to the project's user base.
+
+2. **Create translation files**.
+
+   Inside the "translations" directory, create a new subdirectory for each language you plan to support. Use language codes (e.g., "en" for English, "fr" for French) as directory names.
+
+   ```markdown
+   .
+   └── translations/
+      ├── en/ (English)
+      ├── fr/ (French)
+      └── es/ (Spanish)
+   ```
+
+3. **Translate content**.
+
+   - For each language directory, create translated versions of the documentation files. Typically, you translate Markdown files but consider other formats as needed.
+   - Maintain the same file names and structure as in the original documentation but with translated content.
+
+4. **Update links**.
+
+   In the translated files, ensure that any internal links (e.g., links to other sections or pages within the documentation) are updated to point to the corresponding translated content.
+
+5. **Add language selector**.
```

Comment:
What does it mean by language selector? Is it the flag icon?
If so, should we also mention how to add it here?

Reply:
We have a drop down menu at the moment, and in the Brazilian-PR, we added a flag icon. Check it out:

Reply:
We also have English and French flags in the dropdown menu. Should we add the instructions on how to add the menu here?

Reply:
That would be helpful @adiati98